docs/google/devices.md

```diff
@@ -84,6 +84,44 @@ using a error rate of 0.5% per gate, a circuit of depth 20 and width 20 could be
 at 0.995^(20*20) = 0.135. Using separate error rates per gates (i.e. based on calibration
 metrics) or a more complicated noise model can result in more accurate error estimation.
 
+### Use sweeps when possible
+
+Round trip network time to and from the engine typically adds latency on the order of a second
+to the overall computation time.  Reducing the number of trips and allowing the engine to
+properly batch circuits can improve the throughput of your calculations.  One way to do this
+is to use parameter sweeps to send multiple variations of a circuit at once.
+
+One example is to turn single-qubit gates on or off by using parameter sweeps.  
+For instance, the following code demonstrates combining mesauring the |0> and the |1> state
```

Comment:
"combining measuring" reads strangely. How about "the following code creates a single circuit that parameterizes measurement of the |0> and the |1> state"?

Reply:
Used your suggestion.  Thanks.